Source/CombatExtended/CombatExtended/AmmoInjector.cs

```diff
@@ -290,7 +297,7 @@ private static void ToggleRecipeOnBench(RecipeDef recipeDef, ThingDef benchDef,
                     }
                 }
             }
-            benchDef.allRecipesCached = null;  // Set ammoCraftingStation.AllRecipes to null so it will reset
+            _allRecipesCached.SetValue(benchDef, null);  // Set ammoCraftingStation.AllRecipes to null so it will reset
```

Comment:
We're using a publicized assembly.  Why not set this directly? (`benchDef.allRecipesCached=null`).